tests/LinAlg/vectorTests.hpp

```diff
@@ -1527,19 +1530,80 @@ class VectorTests : public TestBase
     return reduceReturn(fail, &x);
   }
 
+  /// Returns element _i_ of vector _x_.
+  real_type getLocalElement(hiop::hiopVector* x, local_ordinal_type i)
```

Comment:
I ended up removing the const specifier for `hiop::hiopVector* x` in attempting to move this function to `vectorTests.hpp`. Since this is mainly a wrapper for getLocalData and is only used in specific situations, this seems fine to me. If not, I can easily move this and re-implement for each vector test.